test/test_source_gdal.py

```diff
@@ -104,6 +104,17 @@ def testTileLinearStyleFromGeotiffs():
     _assertImageMatches(image, 'geotiff_style_linear_7_22_51')
 
 
+def testTileStyleMatplotlibColormap():
+    testDir = os.path.dirname(os.path.realpath(__file__))
+    imagePath = os.path.join(testDir, 'test_files', 'rgb_geotiff.tiff')
+    style = json.dumps({'band': 1, 'min': 0, 'max': 100,
+                        'palette': 'jet',  # use a named MPL colormap
```

Comment:
I don't see `jet` in MPL.  Did it get added in your test project by another package?

Reply:
Oh shoot, your right. I have `colorcet` installed which added jet to MPL. I will change this to viridis for the test